PlottingScripts/Basic_Plot.py

```diff
@@ -0,0 +1,128 @@
+"""This is a basic script for plotting an analysis file."""
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+import pdb
+
+from bokeh.plotting import figure, show, output_file
+
+
+"""This script is just the first step in visualizing the data set. This file
+comprises of the method creating an html bokeh plot and a command at the end to
+implement the method
+"""
+
+
+def makeplot(Dataframe, Cols, minvalues):
```

Comment:
We haven't talked about this as a group yet, but I think we should settle on a consistent format for naming functions.  I have been using verb_noun - so for my convention I would name this make_plot().  It doesn't matter what format we choose, let's just talk about it and choose one.